tests/resolvers/Mutation/togglePostPin.spec.ts

```diff
@@ -199,4 +201,56 @@ describe("resolvers -> Mutation -> togglePostPin", () => {
     expect(currentPostIsPinned).toBeFalsy();
     expect(updatedPost?.pinned).toBeFalsy();
   });
+
+  it("throws error if title is not provided to pin post", async () => {
+    const { requestContext } = await import("../../../src/libraries");
+    vi.spyOn(requestContext, "translate").mockImplementationOnce(
+      (message) => message
+    );
+    try {
```

Comment:
1. Pinned posts can only be created by Super Admins or Admins for the organization where the post was created.
2. Please make sure this logic is enforced